Wsl.psm1

```diff
@@ -272,6 +272,34 @@ function Get-WslDistribution
     }
 }
 
+<#
+.EXTERNALHELP
+Wsl-help.xml
+#>
+# Retrieves listing of distributions available from online sources
+function Get-WslDistributionOnline {
+    [CmdletBinding()]
+    param(
+    )
+
+    process {
+        $store = $false
+        Invoke-Wsl "--list", "--online" -IgnoreErrors | Select-Object -Skip 1 | ForEach-Object {
+            $name, $friendlyName = $_ -split ' ', 2
+            if ($store -eq $true ) {
+                $friendlyName = $($friendlyName.Split(@(" "), [System.StringSplitOptions]::RemoveEmptyEntries)) -join " "
+                [PSCustomObject]@{
+                    "Name"         = $name
+                    "FriendlyName" = $friendlyName
+                }
+            }
+            if ( $name -contains "NAME" ) {

```

Comment:
Please follow the style of the rest of the file (no space inside the parentheses, and add a blank line after the closing brace).

Additionally, this seems safe because the "NAME" string doesn't appear to be localized, but we should have a comment about that. The text before it is localized, so better to use a case-sensitive exact match to make sure it's not in that text.

```suggestion

            # The "NAME", "FRIENDLY NAME" header is not localized so can be used to find the start
            # of the list
            if ($name -ceq "NAME") {
```

Reply:
Actually, make this an `elseif` since there's no need to do this check if $store is already true.

Reply:
Updated and added the comments just below the "elseif" start code statement.  Will do some testing with "-ceq" as well.  I think I tried using "-eq" and that did not match.

Reply:
This seems to work based on testing.